addons/web/static/src/views/form/form_renderer.js

```diff
@@ -31,6 +33,10 @@ export class FormRenderer extends Component {
         useBounceButton(useRef("compiled_view_root"), () => {
             return !record.isInEdition;
         });
+        this.uiService = useService('ui');
+        this.onResize = _.debounce(this.render.bind(this), 200);
```

Comment:
prefer `useDebounced(this.render, 200)` (automatically bound to component, cancels any pending debounced call onWillUnmount)